nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/resources/docs/org.apache.nifi.lookup.RestLookupService/additionalDetails.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>RestLookupService</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+    <h2>General</h2>
+    <p>This lookup service has the following required keys:</p>
+    <ul>
+        <li>mime.type</li>
+        <li>request.method; valid values:
+            <ul>
+                <li>delete</li>
+                <li>get</li>
+                <li>post</li>
+                <li>put</li>
+            </ul>
+        </li>
+    </ul>
+    <p>In addition to the required keys, a key "body" can be added which contains a string representing JSON, XML, etc. to be sent with any
+    of those methods except for "get."</p>
+    <p>The record reader is used to consume the response of the REST service call and turn it into one or more records. The record path property
+    is provided to allow for a lookup path to either a nested record or a single point deep in the REST response. Note: a valid schema must be
+    built that encapsulates the REST response accurately in order for this service to work.</p>
+    <h2>Headers</h2>
+    <p>Headers are supported using dynamic properties. Just add a dynamic property and the name will be the header name and the value will be the value for the header. Expression language
+    powered by input from the variable registry is supported.</p>
+    <h2>Dynamic URLs</h2>
+    <p>The URL property supports expression language in a non-standard way: through the lookup key/value pairs configured on the processor. The configuration specified by the user will be passed
```

Comment:
I'd omit the expression, 'in a non-standard way', because I envision this pattern will be used more frequently and I'm planning to add copy FlowFile attribute to coordinates capability at LookupRecord processor

Reply:
Done.